### PR TITLE
New Package: python-keyutils-0.4_1

### PR DESCRIPTION
**Pull request by mfulz**

### CODE REVIEW

File: srcpkgs/python-keyutils/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
+	}
+}
+python3.4-keyutils_package() {
+	build_style=meta
+	short_desc="${short_desc/Python2/Python3.4} (transitional dummy package)"
+	depends="python3-keyutils>=${version}_${revision}"
+}
```
Comment (Vaelatern) [inline srcpkgs/python-keyutils/template]: Please remove this transition package entirely. The transition packages exist only for reasons of easing updates of packages that used to be named things like `python3.4-*`. Does not apply to new packages.

File: srcpkgs/python-keyutils/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+short_desc="A set of python bindings for keyutils"
+maintainer="Matthias Fulz <mfulz@olznet.de>"
+homepage="https://github.com/sassoftware/python-keyutils"
+license="Apache"
+distfiles="https://github.com/sassoftware/python-keyutils/archive/v$version.tar.gz"
+checksum=c452b6f387443586d5737a5969d8fe044d443b98f1f1440a65610e9caf5bcd6a
+
+python3-keyutils_package() {
+	pycompile_module="keyutils"
+	replaces="python3.4-keyutils>=0"
```
Comment (Vaelatern) [inline srcpkgs/python-keyutils/template]: And remove this reference to the transition package.

File: srcpkgs/python-keyutils/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	short_desc="${short_desc/Python2/Python3}"
```
Comment (Vaelatern) [inline srcpkgs/python-keyutils/template]: Totally ineffective. Please alter your `short_desc` to make this actually change stuff.

File: srcpkgs/python-keyutils/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+pkgname=python-keyutils
+version=0.4
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="keyutils"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel keyutils-devel"
+depends="python-setuptools"
+short_desc="A set of python bindings for keyutils"
```
Comment (Vaelatern) [inline srcpkgs/python-keyutils/template]: `Python2 bindings for keyutils`

File: srcpkgs/python-keyutils/template [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'python-keyutils'
+pkgname=python-keyutils
+version=0.4
+revision=1
+noarch=yes
```
Comment (Vaelatern) [inline srcpkgs/python-keyutils/template]: This is not `noarch`. Remove this line

**Comment by mfulz** [comment]
Changes applied.

**Pull request merged**